org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/Tracker.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Objects;
 import java.util.function.Function;
 import javax.annotation.Nullable;
+import org.sonarlint.eclipse.core.internal.utils.StringUtils;
 
 // TODO this entire class should be shared with IntelliJ
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)